src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -319,10 +231,10 @@
 type: command
 short-summary: Place the CLI in a waiting state until a deployment condition is met.
 examples:
-  - name: Place the CLI in a waiting state until a deployment condition is met. (autogenerated)
```

Comment:
@zhoxing-ms , please note that all `(autogenerated)` and `crafted: true` have been removed.  Telling our customers that a sample was autogenerated is not helpful to them.

Reply:
I'm not sure whether the original design has other purposes besides informing users that these examples are automatically generated by Aladdin. @kairu-ms What do you think of this?

Reply:
@kairu-ms, am I good to go with this type of _help.py file update?